backend/canvas_app_explorer/models.py

```diff
@@ -40,6 +40,7 @@ class LtiTool(models.Model):
     support_resources = HTMLField()
     canvas_placement = models.ManyToManyField(CanvasPlacement)
     internal_notes = HTMLField(blank=True, null=True, help_text="a place to put helpful info for admins, not visible to users")
+    launch_url = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Not sure if we'll have to have long URL's but should _probably_ match the URL Length limit of 2048

Reply:
updated to 2028

Reply:
This also needs to be updated in the model here.

Also looking at this we should add help_text which would display for the admin. I'm trying to remember if the idea was that either the canvas_id OR the launch_url be added? What if there's values in both?

Reply:
In testing, if `launch_url` is defined it doesn't matter what value is in `canvas_id`

So maybe help_text for launch_url should be something like.

"A link that will directly be launched by clicking on this card. If this is value is set then canvas_id is ignored"

Reply:
help_text added!